Code/Engine/RendererVulkan/Device/Implementation/DeviceVulkan.cpp

```diff
@@ -39,28 +39,53 @@
 #  include <GLFW/glfw3.h>
 #endif
 
-VKAPI_ATTR VkBool32 VKAPI_CALL debugCallback(VkDebugUtilsMessageSeverityFlagBitsEXT messageSeverity, VkDebugUtilsMessageTypeFlagsEXT messageType, const VkDebugUtilsMessengerCallbackDataEXT* pCallbackData, void* pUserData)
+EZ_DEFINE_AS_POD_TYPE(VkLayerProperties);
+
+namespace
 {
-  switch (messageSeverity)
+  VKAPI_ATTR VkBool32 VKAPI_CALL debugCallback(VkDebugUtilsMessageSeverityFlagBitsEXT messageSeverity, VkDebugUtilsMessageTypeFlagsEXT messageType, const VkDebugUtilsMessengerCallbackDataEXT* pCallbackData, void* pUserData)
   {
-    case VK_DEBUG_UTILS_MESSAGE_SEVERITY_VERBOSE_BIT_EXT:
-      ezLog::Debug("VK Verbose: {}", pCallbackData->pMessage);
-      break;
-    case VK_DEBUG_UTILS_MESSAGE_SEVERITY_INFO_BIT_EXT:
-      ezLog::Info("VK Info: {}", pCallbackData->pMessage);
-      break;
-    case VK_DEBUG_UTILS_MESSAGE_SEVERITY_WARNING_BIT_EXT:
-      ezLog::Warning("VK warning: {}", pCallbackData->pMessage);
-      break;
-    case VK_DEBUG_UTILS_MESSAGE_SEVERITY_ERROR_BIT_EXT:
-      ezLog::Error("VK error: {}", pCallbackData->pMessage);
-      break;
-    default:
-      break;
+    switch (messageSeverity)
+    {
+      case VK_DEBUG_UTILS_MESSAGE_SEVERITY_VERBOSE_BIT_EXT:
+        ezLog::Debug("VK Verbose: {}", pCallbackData->pMessage);
```

Comment:
That it is a "verbose / info / error" message, is already covered by the ezLog function that you are calling. I would only prefix it with "VK:" or "Vulkan:"

Reply:
I did not write that code. Only it's intendation changed. Should I still change the message as part of this PR?

Reply:
Interesting. I leave that up to you.